installer/dev/install.cpp

```diff
@@ -191,9 +189,22 @@ namespace WindowsAppRuntimeInstaller
             return true;
         }
 
-        // On Arm64 systems, all current package architectures are applicable.
+        // On Windows 11 (i.e. builds 22000+) ARM64 systems, all framework package architectures are applicable.
+        // On Windows 10 (i.e. builds 17763-190**) ARM64 systems (which don't support X64 apps), all but x64 framework package architecture is not applicable.
         if (systemArchitecture == ProcessorArchitecture::Arm64)
         {
+            if (packageProperties->architecture == ProcessorArchitecture::X64)
+            {
+                if (WindowsVersion::IsWindows11_21H2OrGreater())
+                {
+                    return true;
+                }
+                else
+                {
+                    return false;
+                }
```

Comment:
Replace lines 198-205 with `return WindowsVersion::IsWindows11_21H2OrGreater();`

No need for the if ...return true else return false.